### PR TITLE
Add EMD metric to training scripts

### PR DESCRIPTION
**Pull request by therwig**
Adding EMD as calculated from the OT package. Also includes a test script to verify that OT is installed and gives sensible results for a simple input data example

### CODE REVIEW

File: qTrain.py [hunk 1]
```diff
@@ -12,12 +13,15 @@
...
+import ot
+
 @numba.jit
 def normalize(data):
     norm =[]
     for i in range(len(data)):
-        norm.append( data[i].max() )
-        data[i] = 1.*data[i]/data[i].max()
+        norm.append( data[i].sum() )
+        data[i] = 1.*data[i]/data[i].sum()
```
Comment (kakwok) [inline qTrain.py]: may want to turn this into an option, could do later

Comment (therwig) [reply qTrain.py]: Agreed! let me add a flag

File: train.py [hunk 2]
```diff
@@ -364,7 +404,8 @@ def trainCNN(options,args):
     if not os.path.exists(model_name):
       os.mkdir(model_name)
     os.chdir(model_name)
-   
+    shutil.copyfile("../../tests/verify_emd.py", "verify_emd.py")
```
Comment (kakwok) [inline train.py]: @therwig Is this needed?  or just for testing?

Comment (therwig) [reply train.py]: its not needed anymore. was a test/hack. I'll remove it

**Comment by therwig** [comment]
Pushed fixes to the above comments

**Comment by kakwok** [comment]
Thanks! Awesome!

**Pull request merged**